lib/elixir/lib/supervisor.ex

```diff
@@ -399,15 +399,19 @@ defmodule Supervisor do
   So far we have started the supervisor passing a single child as a tuple
   as well as a strategy called `:one_for_one`:
 
-      Supervisor.start_link([
+      children = [
         {Stack, [:hello]}
-      ], strategy: :one_for_one)
+      ]
+
+      Supervisor.start_link(children, strategy: :one_for_one)
```

Comment:
I considered
```elixir
children = [{Stack, [:hello]}]
Supervisor.start_link(children, strategy: :one_for_one)
```
but this way it's more consistent with existing well-formatted examples